### ISSUE TITLE
Log unauthorized repo access error on warning level

### ISSUE THREAD

**Issue by sslavic** [sbt/sbt#1218]
Currently (sbt 0.13.2-RC1), when a repository which requires authorization is used to resolve dependencies, but credentials are not configured for that repository, sbt will just log on warning level that a repository has been tried, that dependency could not be resolved, and build will fail. Only when turning on debug logging level one can see that there was authorization problem with message like:

```
[debug] CLIENT ERROR: Unauthorized url=https://repo.foo.com/libs-snapshot-local/com/foo/bar_2.10/0.0.1-SNAPSHOT/maven-metadata.xml
```

Please have this message logged on warning level.

**Comment by jsuereth** [comment]
hmm, this shouldn't the case, I think.  Thanks for reporting.

**Comment by jsuereth** [comment]
i.e. we just added notifications on authentication failures....

Can you show me the full update output?

**Comment by jsuereth** [comment]
Hmm, looks like the commit to fix this wasn't merged into 0.13.2.  Can you try 0.13.5-M2 and see if that fixes the issue?  

Thanks!

**Comment by sslavic** [comment]
Yes, with 0.13.5-M2 it's much better:

```
[error] Unable to find credentials for [Artifactory Realm @ repo.foo.com].
```

Issue can be closed. Thanks!

**Comment by jsuereth** [comment]
cc @eed3si9n Do you think we should get that fix in 0.13.2-RC1?  It was intended for that I think the branch-shuffling killed it.

My current vote is that 0.13.5 will be out soon enough...

**Comment by eed3si9n** [comment]
+1 for cherry picking 21da87c4e7df3904ab5dbdcd5d5bdfa53102e86f into 0.13.2 if that's considered a bug fix.

**Comment by eed3si9n** [comment]
If that's going to delay 0.13.2 for another two weeks by pushing the fix in, then @jsuereth could be right about waiting till 0.13.5.

**Comment by jsuereth** [comment]
Yeah, I'd rather not delay 0.13.2 further.  Let's just have the fix in 0.13.5

**Comment by sslavic** [comment]
Not a blocker for me, so +1 for delay, and also +1 for more frequent minor releases.
